java/operator/src/main/java/com/instaclustr/cassandra/operator/controller/DataCenterReconciliationController.java

```diff
@@ -157,7 +157,15 @@ private void createOrReplaceStateNodesStatefulSet(final Iterable<ConfigMapVolume
                     .addVolumeMountsItem(new V1VolumeMount()
                             .name("pod-info")
                             .mountPath("/etc/podinfo")
-                    );
+                    )
+                    .addVolumeMountsItem(new V1VolumeMount()
+                            .name("sidecar-config-volume")
+                            .mountPath("/tmp/sidecar-config-volume")
+                    )
+                    .addArgsItem("/tmp/sidecar-config-volume")
+                    .addEnvItem(new V1EnvVar().name("NAMESPACE").valueFrom(new V1EnvVarSource().fieldRef(new V1ObjectFieldSelector().fieldPath("metadata.namespace"))))
```

Comment:
Can we project these into the pod via the Downward API instead of ENV vars? https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/

We are trying to keep all information exposed to the pods about Kubernetes to be done via downward api rather than ENV vars.

Reply:
These ENV vars were here before the PR. I only added the `argsItem` and `mountPath`